### ISSUE TITLE
Required field prompt not displaying on mobile browser

### ISSUE THREAD

**Issue by annaballasiotes** [Gig-o-Matic/GO3#290]
Noticed when changing status of a gig; on phone, clicking "Save" was not working but there was no visible prompt as to why; when trying on Desktop, a pop-up prompted to correct an item / choose an option from the list to fill in "Contact." Perhaps make "Contact" not required, or if possible, migrate "Contact" over from old gig-o?

**Comment by aaronoppenheimer** [comment]
Is this for a gig that already existed and was imported? When creating a new gig, the contact is filled in automatically - it can't be blank. But for an imported gig, the code probably doesn't deal well with blank contact because it's not supposed to be possible. Once all the imported gigs are updated, this should no longer be a problem.

Ben, I can export the contact member but the database ID won't translate.

Anna, if you create a new gig on mobile and save it without filling in a title for the gig, do you see the pop-up prompt asking you to fill it in?

**Comment by annaballasiotes** [comment]
Yes, the prompt works in that scenario on mobile! I think then this bug is only for the imported gigs.

**Comment by bklang** [comment]
Thanks for the diagnosis @aaronoppenheimer . I'll modify the importer code to grab the first band admin and make them the contact. It might not be correct, but at least people won't get stuck.
